src/routes/(app)/checkout/+page.server.ts

```diff
@@ -373,7 +374,13 @@ export const actions = {
 				delete billingInfo.billing.vatNumber;
 			}
 		}
-
+		const desiredPayment = z
+			.object({
+				paymentTimeOut: z.number({ coerce: true }).optional()
```

Comment:
Make it `min(1)` and positive integer

Reply:
min(1) gives error if the param is not changed